tools/scripts/AksEdgeQuickStart/AksEdgeQuickStartForAio.ps1

```diff
@@ -0,0 +1,264 @@
+$azcfg = (Get-AideUserConfig).Azure
+if ($azcfg.Auth.Password) {
+   Write-Host "Password found in json spec. Skipping AksEdgeAzureSetup." -ForegroundColor Cyan
+   $skipAzureArc = $false
+} elseif ($skipAzureArc) {
+    Write-Host ">> skipping step 2" -ForegroundColor Yellow
+} else {
+    $aksedgeazuresetup = (Get-ChildItem -Path "$workdir" -Filter AksEdgeAzureSetup.ps1 -Recurse).FullName
+    & $aksedgeazuresetup -jsonFile $aidejson -spContributorRole -spCredReset
+
+    if ($LastExitCode -eq -1) {
+        Write-Host "Error in configuring Azure Cloud for Arc connection" -ForegroundColor Red
+        Stop-Transcript | Out-Null
+        Pop-Location
+        exit -1
+    }
+}
+
+# Download, install and deploy AKS EE 
+Write-Host "Step 3: Download, install and deploy AKS Edge Essentials"
+# invoke the workflow, the json file already updated above.
+$retval = Start-AideWorkflow -jsonFile $aidejson
+Write-Host "Return: $retval"
+if ($retval) {
+    Write-Host "Deployment Successful. "
+} else {
+    Write-Host -Message "Deployment failed" -Category OperationStopped -ForegroundColor Red
+    Stop-Transcript | Out-Null
+    Pop-Location
+    exit -1
+}
+
+Write-Host "Step 4: Connect to Arc"
+if ($skipAzureArc) {
+    Write-Host ">> skipping step 4" -ForegroundColor Yellow
+} else {
+    Write-Host "Installing required Az Powershell modules"
+    $arcstatus = Initialize-AideArc
+    if ($arcstatus) {
+        Write-Host ">Connecting to Azure Arc"
+        if (Connect-AideArc) {
+            Write-Host "Azure Arc connections successful."
+        } else {
+            Write-Host "Error: Azure Arc connections failed" -ForegroundColor Red
+            Stop-Transcript | Out-Null
+            Pop-Location
+            exit -1
+        }
+    } else { Write-Host "Error: Arc Initialization failed. Skipping Arc Connection" -ForegroundColor Red }
+}
+
+Write-Host "Step 5: Prep for AIO workload deployment"
+Write-Host "Deploy local path provisioner"
+try {
+    & 'C:\Program Files\AksEdge\kubectl\kubectl.exe' apply -f 'https://raw.githubusercontent.com/Azure/AKS-Edge/main/samples/storage/local-path-provisioner/local-path-storage.yaml' 
+    Write-Host "Successfully deployment the local path provisioner"
+}
+catch {
+    Write-Host "Error: local path provisioner deployment failed" -ForegroundColor Red
+    Stop-Transcript | Out-Null
+    Pop-Location
+    exit -1 
+}
+
+Write-Host "Configuring firewall specific to AIO"
+try {
+    New-NetFirewallRule -DisplayName "AIO MQTT Broker" -Direction Inbound -Action Allow | Out-Null
+    Write-Host "Successfully added firewall rule for AIO MQTT Broker"
+}
+catch {
+    Write-Host "Error: Firewall rule addition for AIO MQTT broker failed" -ForegroundColor Red
+    Stop-Transcript | Out-Null
+    Pop-Location
+    exit -1 
+}
+
+Write-Host "Configuring port proxy for AIO"
+try {
+    netsh interface portproxy add v4tov4 listenport=1883 listenaddress=0.0.0.0 connectport=1883 connectaddress=192.168.0.4 | Out-Null
```

Comment:
192.168.0.4 -> This is the address if the 192.168.0.X family is available,. but could change - Probably we need to check and use the appropriate subnet

Reply:
Let me check how we pick subnet in AKS-EE and see if we can extend the same logic here.

Reply:
Using the `Get-AksEdgeDeploymentInfo` command to get the service ip start and use it.